feature/home/src/main/java/org/sopt/home/AddFriendBottomSheet.kt

```diff
@@ -0,0 +1,96 @@
+package org.sopt.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.designsystem.component.button.RegularButton
+import org.sopt.designsystem.component.textfield.RegularTextField
+import org.sopt.designsystem.ui.theme.NOWSOPTAndroidTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AddFriendBottomSheet(
+    modifier: Modifier = Modifier,
+    sheetState: SheetState,
+    onDismissRequest: () -> Unit = {},
+    onClickSave: () -> Unit,
+    onNameValueChange: (String) -> Unit,
+    onHobbyValueChange: (String) -> Unit,
+    state: HomeState,
+) {
```

Comment:
안드로이드(컴포즈) 코딩 컨벤션인데 참고만 하세요.

- required parameter는 optional parameter 위에 있어야 합니다.
- Modifier는 모든 optional parameter 중 가장 상위에 있어야 합니다.

그렇다면 아래와 같이 바꿀 수 있습니다.

```suggestion
fun AddFriendBottomSheet(
    sheetState: SheetState,
    onClickSave: () -> Unit,
    onNameValueChange: (String) -> Unit,
    onHobbyValueChange: (String) -> Unit,
    state: HomeState,
    modifier: Modifier = Modifier,
    onDismissRequest: () -> Unit = {},
) {
```

그리고 state가 많이 주입되고 있는데, state 호이스팅을 이렇게 하는 이유가 있을까요?

Reply:
컴포즈 지식이 짧아서 컨벤션도 잘 모르고 있었는데, 감사합니다 ! 

그리고 상태 호이스팅에 아직 익숙하지 않아서 무작정 끌어올리기만 하고 있었습니다. 일종의 패턴 중독이었던 것 같습니다. 좀 더 재사용성과 적합성등을 고려해서 불필요한 호이스팅은 피하도록 하겠습니다 !